FFXIVClientStructs/FFXIV/Client/UI/Misc/RaptureHotbarModule.ClearCallback.cs

```diff
@@ -0,0 +1,38 @@
+namespace FFXIVClientStructs.FFXIV.Client.UI.Misc;
+
+public partial struct RaptureHotbarModule {
+    [StructLayout(LayoutKind.Explicit, Size = 0x08)]
+    public unsafe partial struct ClearCallback {
+        [FieldOffset(0x0)] public void** vtbl;
```

Comment:
You should add a `, CExportIgnore()` for this, since it's exported from the VirtualFunction generator to the header files